src/js_native_api.h

```diff
@@ -5,10 +5,11 @@
 #include <stdbool.h>
 #include "js_native_api_types.h"
 
+#define NAPI_VERSION_EXPERIMENTAL 2147483647
 #ifndef NAPI_VERSION
 #ifdef NAPI_EXPERIMENTAL
 // Use INT_MAX, this should only be consumed by the pre-processor anyway.
```

Comment:
Should this comment be also updated/moved?

Reply:
good point, will update